livre/eq/Node.cpp

```diff
@@ -119,17 +119,36 @@ class Node
         _textureDataCachePtr.reset();
     }
 
-    void updateAndSendFrameRange()
+    void getCurrentPrevFrameRanges( Vector2ui& prevRange,
+                                    Vector2ui& currentRange )
     {
-        if( !_dataSourcePtr )
-            return;
-
+        const livre::VolumeInformation& info =
+                _dataSourcePtr->getVolumeInformation();
+        prevRange = info.frameRange;
         _dataSourcePtr->update();
+        currentRange = info.frameRange;
+    }
+
+    void sendFrameRange()
+    {
         const livre::VolumeInformation& info =
                 _dataSourcePtr->getVolumeInformation();
+
         _config->sendEvent( VOLUME_FRAME_RANGE ) << info.frameRange;
     }
 
+    void updateAndSendFrameRange()
+    {
+        if( !_dataSourcePtr )
+            return;
+
+        Vector2ui prevRange;
+        Vector2ui currentRange;
+        getCurrentPrevFrameRanges( prevRange, currentRange );
+        if( currentRange != prevRange )
```

Comment:
I'm testing and I need this here
"if( currentRange != prevRange || currentRange == FULL_FRAME_RANGE )"
to have the default 'livre' working